src/cdomains/intDomain.ml

```diff
@@ -779,6 +804,66 @@ struct
     in
     QCheck.(set_shrink shrink @@ set_print show @@ map (*~rev:BatOption.get*) (of_interval ik) pair_arb)
   let relift x = x
+
+  let abs x = let r = Ints_t.neg x
+   in if Ints_t.compare x r < 0 then r else x
```

Comment:
Is it just me or is this implementation of `abs` a bit weird? The following seems more natural and common:
```ocaml
let abs x =
  if Ints_t.compare x Ints_t.zero < 0 then
    Ints_t.neg x
  else
    x
```

Reply:
Yeah, I stumbled on the same thing, I think it somehow saves one operation in the true case but lets' use your suggestion that seems simpler